### PR TITLE
Feat/dtcore 2668

### PR DESCRIPTION
**Pull request by sidlynx**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/pci-load-balancer`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | DTCORE-2666 & DTCORE-2668
| License          | BSD 3-Clause

- [ ] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Review by kqesar — APPROVED** [review]

File: packages/manager/apps/pci-load-balancer/src/api/hook/useLoadBalancer.tsx [hunk 1]
```diff
@@ -331,3 +333,47 @@ export const useLoadBalancerListeners = (
...
+        floatingIp,
+        privateNetwork,
+        subnet,
+        gateways,
+        listeners,
+        name,
+      }),
+    onError,
+    onSuccess: async () => {
+      // TDOO invalidate right query
```
Comment (frenautvh) [inline packages/manager/apps/pci-load-balancer/src/api/hook/useLoadBalancer.tsx:368]: will it be done in another PR ?

Comment (sidlynx) [reply packages/manager/apps/pci-load-balancer/src/api/hook/useLoadBalancer.tsx:368]: yep i forgot to invalidate the query, hopefully i left the TODO there, will fix it in the TUs task

**Review by seven-amid — APPROVED** [review]

**Review by frenautvh — APPROVED** [review]
